### PR TITLE
Generate changelog for 3.0.0-alpha2

### PR DESCRIPTION
**Pull request by kyukhin**
NO_DOC=no code changes
NO_TEST=no code changes
NO_CHANGELOG=no code changes

### CODE REVIEW

File: changelogs/3.0.0-alpha2.md [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+  to future changes (gh-6493).
+* Introduced the new `IPROTO_WATCH_ONCE` request to get the value currently
+  associated with a notification key on a remote instance without subscribing
+  to future changes (gh-6493).
+* Introduced the `conn:watch_once()` net.box connection method to get the value
+  currently associated with a notification key on a remote instance without
+  subscribing to future changes. The new method is implemented using the
+  `IPROTO_WATCH_ONCE` request type (gh-6493).
+* Introduced the default field values in the space format (gh-8157).
+* Added support for functional variant of the default field values (gh-8609).
```
Comment (andreyaksenov) [inline changelogs/3.0.0-alpha2.md:58]:
```suggestion
* Added support for a functional variant of the default field values (gh-8609).
```

File: changelogs/3.0.0-alpha2.md [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+* The default directory and file paths are changed to `var/run/<...>`,
+  `var/log/<...>`, `var/lib/<...>` and so on (gh-8862).
+* The directory that contains the configuration file is now added into the Lua
+  modules search paths (gh-8862).
+* Most of vshard options are now added in the config (gh-9007).
+* Added initial support of vshard (gh-9007).
+
+### Console
+
+* Now multiline commands can be used by setting continuation marker
```
Comment (andreyaksenov) [inline changelogs/3.0.0-alpha2.md:282]:
```suggestion
* Now multiline commands can be used by setting the continuation marker
```

File: changelogs/3.0.0-alpha2.md [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+* An `IPROTO_EVENT` packet now has the same sync number as the last
+  corresponding `IPROTO_WATCH` request (gh-8393).
+* Fixed a bug because of which a dirty (not committed to WAL) DDL record could
+  be written to a snapshot and cause a recovery failure (gh-8530).
+* **[Breaking change]** The key `box.space._schema['cluster']` is renamed to
+  `'replicaset_uuid'`. That is not expected to be breaking because `_schema` is
+  an internal system space, but the key was visible in public and documented
+  (gh-5029).
+* **[Breaking change]** The table `box.info.cluster` is renamed to
+  `box.info.replicaset`. The behaviour can be reverted using the `compat` option
```
Comment (andreyaksenov) [inline changelogs/3.0.0-alpha2.md:341]:
```suggestion
  `box.info.replicaset`. The behavior can be reverted using the `compat` option
```

File: changelogs/3.0.0-alpha2.md [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+* Fixed incorrect conversion of an integer greater than `INT64_MAX` or
+  less than `0` to a decimal number during SQL arithmetic operations (gh-8460).
+* Fixed an assertion when selecting tuples with incomplete internal
+  format (gh-8418).
+* Fixed integer overflow issues in built-in functions (ghs-119).
+* Fixed a Use-After-Free vulnerability in the `WITH RECURSIVE` clause
+  (ghs-119).
+* Fixed a possible assertion or segmentation fault when optimizing
+  `INSERT INTO ... SELECT FROM` (gh-8661).
+* Fixed an integer overflow issue and added check for the `printf()` failure due
```
Comment (andreyaksenov) [inline changelogs/3.0.0-alpha2.md:505]:
```suggestion
* Fixed an integer overflow issue and added a check for the `printf()` failure due
```

File: changelogs/3.0.0-alpha2.md [hunk 5]
```diff
@@ -0,0 +1,577 @@
...
+* Fixed the privilege check when using spaces with functional indexes and
+  constraints (gh-7873).
+* Fixed the inability to insert an integral number into a double-formatted
+  field (gh-7483).
+* Fixed a bug in the box console implementation because of
+  which the `language` parameter was shared between connected
+  clients (gh-8817).
+* Fixed the invalid memory access in a corner case of a specialized comparison
+  function (gh-8899).
+* Fixed a crash when `box.iproto.override` was called with unconfigured box.
```
Comment (andreyaksenov) [inline changelogs/3.0.0-alpha2.md:538]:
```suggestion
* Fixed a crash when `box.iproto.override` was called with an unconfigured box.
```

**Review by andreyaksenov — APPROVED** [review]

File: changelogs/3.0.0-alpha2.md (hunk 1 above)
Comment (kyukhin) [reply changelogs/3.0.0-alpha2.md]: Fixed.

File: changelogs/3.0.0-alpha2.md (hunk 2 above)
Comment (kyukhin) [reply changelogs/3.0.0-alpha2.md]: Fixed.

File: changelogs/3.0.0-alpha2.md (hunk 4 above)
Comment (kyukhin) [reply changelogs/3.0.0-alpha2.md]: Fixed.

**Pull request merged**